src/components/NavMenu/index.tsx

```diff
@@ -43,15 +46,29 @@ export const NavMenu = () => {
                                     neil.sims@flowbite.com
                                 </p>
                             </div>
-                            <ul className="py-1" role="none">
+                            <ul className="xpy-1" role="none">
```

Comment:
오타 나신 것 같아요! 수정 부탁드립니다.

```suggestion
  <ul className="py-1" role="none">
```